core/src/main/scala/kafka/coordinator/group/GroupMetadataManager.scala

```diff
@@ -1139,6 +1139,7 @@ object GroupMetadataManager {
 
   private val CURRENT_OFFSET_KEY_SCHEMA = schemaForKey(CURRENT_OFFSET_KEY_SCHEMA_VERSION)
   private val CURRENT_GROUP_KEY_SCHEMA = schemaForKey(CURRENT_GROUP_KEY_SCHEMA_VERSION)
+  private val CURRENT_GROUP_METADATA_VALUE_SCHEMA_VERSION = GROUP_VALUE_SCHEMAS.size - 1
```

Comment:
This would be more robust if we did something like:

```scala
GROUP_VALUE_SCHEMAS.keySet.max
````

Or something along those lines.

Reply:
good suggestion! Thanks.